denizen_scripts/survival/repo-link/cuboid_creator.dsc

```diff
@@ -33,14 +37,14 @@ cuboid_command:
   permission message: <&c>Sorry <player.name>, you can't perform that command.
   script:
```

Comment:
this script is both:
- missing a fallback in the case that the first argument is not empty, `clear`, `save`, or `schem`
- is checking static arguments - would recommend a `choose`/`case` set here instead